splitio/client/input_validator.py

```diff
@@ -353,9 +429,7 @@ def validate_manager_feature_flag_name(feature_flag_name, should_validate_exista
     :return: feature_flag_name
     :rtype: str|None
     """
-    if (not _check_not_null(feature_flag_name, 'feature_flag_name', 'split')) or \
-       (not _check_is_string(feature_flag_name, 'feature_flag_name', 'split')) or \
-       (not _check_string_not_empty(feature_flag_name, 'feature_flag_name', 'split')):
+    if not _validate_manager_feature_flag_name(feature_flag_name):
```

Comment:
As a suggestion, you can call `_validate_feature_flag_name(feature_flag_name, 'split')`, and remove `_validate_manager_feature_flag_name` function.

Reply:
will do